### ISSUE TITLE
ignore-warning feature request

### ISSUE THREAD

**Issue by sushant-pradhan** [hadolint/hadolint#274]
- [x] This is a feature request
- [x] I searched existing issues before opening this one

### Expected behavior
I searched the existing issues but didn't find anything relevant. The linting feature is really useful but I feel if we have a ignore-warning ability it will be useful. In my CI pipeline (running on Jenkins), I am linting Dockefile and it fails (exit status non zero) if there are linting issues. What I would like to rather do is to print these as warnings/email the committer and still proceed with my build. So an additional flag which can just print the warnings but exit with a 0 status (success code) would be useful

```bash
$ hadolint Dockerfile; echo $?
0
$ hadolint --sandbox-mode Dockerfilewithissues; echo $?
Dockerfilewithissues:2 DL3010 Use ADD for extracting archives into an image
0
```

### Actual behavior
```bash
$ hadolint Dockerfile; echo $?
0
$ hadolint Dockerfilewithissues; echo $?
Dockerfilewithissues:2 DL3010 Use ADD for extracting archives into an image
1
```
